frontend/app/composables/setFavicon.ts

```diff
@@ -0,0 +1,16 @@
+export function setFavicon() {
```

Comment:
```suggestion
export function useFavicon() {
```

Rename file accordingly